src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+  Hello, world!
+  </body>
```

Comment:
Did you run `npm run lint`?
Add indent to the text
```suggestion
  <body>
    Hello, world!
  </body>
```

Reply:
ofc,ok.
will check it at q&a today.
<img width="448" alt="Screenshot 2021-11-15 at 14 42 29" src="https://user-images.githubusercontent.com/78903159/141784078-ee9bc632-b251-4db3-abb6-c7bf1c8946ed.png">